src/core/a-animation.js

```diff
@@ -373,6 +379,19 @@ function getAnimationValues (el, attribute, dataFrom, dataTo, currentValue) {
   };
 
   /**
+   * Match the scema type to color
+   * @return {bool} if the schema is of type color
+   */
+  function isColor () {
+    var componentName = attributeSplit[0];
+    var propertyName = attributeSplit[1];
+    var component = el.components[componentName];
+    var schema = component && component.schema;
+    if (schema) { return schema[propertyName].type === 'color'; }
```

Comment:
- tween.js can already tween any object with numeric values.
- Might want to refactor if it gets any more complicated than this, will re-evaluate. Are there any other special cases we need to consider? Colors might be the last big special case.

Code looks OK for this scope. Minimal changes, tests pass, things work :white_check_mark: